node/cn/backend.go

```diff
@@ -873,7 +873,7 @@ func (s *CN) Start(srvr p2p.Server) error {
 	s.startBloomHandlers()
 
 	// Start the RPC service
-	s.netRPCService = api.NewPublicNetAPI(srvr, s.NetVersion())
+	s.netRPCService = api.NewNetAPI(srvr, s.NetVersion())
```

Comment:
@da1suk8 When dealing with RPCs outside `api/`, please check here. Other APIs are all declared in `cn.API()` yet NetAPI is declared here. Please check if we can bring it into API() and remove `s.netRPCService` field.

Reply:
Noted thank you.